requirements/edx/base.in

```diff
@@ -64,7 +64,7 @@ djangorestframework-jwt
 docutils
 dogapi==1.2.1                       # Python bindings to Datadog's API, for metrics gathering
 enum34==1.1.6
-edx-ace
+-e git+https://github.com/appsembler/edx-ace@omar/django-email-channel#egg=edx-ace==0.1.6.omar
```

Comment:
@mulby I'm sure you're aware of this hack. Which we should stop and use proper `pip-tools` method once the edX ACE PR is done.